src/System.Runtime/tests/System/String.cs

```diff
@@ -401,24 +362,26 @@ public static void TestCompareTo()
         Assert.True(i > 0);
     }
 
-    [Fact]
-    public static void TestContains()
+    [Theory]
+    [InlineData("Hello", "ello", true)]
+    [InlineData("Hello", "", true)]
+    [InlineData("Hello", "ELL", false)]
+    [InlineData("Hello", "Larger Hello", false)]
+    [InlineData("Hello", "Goodbye", false)]
```

Comment:
It'd be good to have cases where ```text``` is empty, both where ```value``` is empty and where it isn't.